### DISCUSSION TITLE
new Option: show Region-file

### DISCUSSION THREAD

**Discussion by TjarkTV** [maruohon/minihud#258] [Ideas]
i wonder if its possible to show the region-file, of the chunk in which the player is right now. I need to convert them all the time with this tool: [https://dinnerbone.com/minecraft/tools/coordinates/](https://dinnerbone.com/minecraft/tools/coordinates/ ) maybe its possible to calculate it and display it on a new line for the hud.

**Comment by maruohon** [comment]
There has been a `infoRegionFile` info line for a long time by now. And there is also a `overlayRegionFile` overlay renderer to show the boundaries of the region you are in.
